src/cloudai/report_generator/tool/tensorboard_data_reader.py

```diff
@@ -45,8 +46,8 @@ def extract_data(self, tag: str) -> List[Tuple[int, float]]:
         for root, _, files in os.walk(self.directory_path):
```

Comment:
IMO it makes sense to switch away from `os.walk` to `Path` alternatives here too.